core/drivers/crypto/caam/include/caam_blob.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2020 Pengutronix, Rouven Czerwinski <entwicklung@pengutronix.de>
+ */
+#ifndef __CAAM_BLOB_H__
+#define __CAAM_BLOB_H__
+
+#include <caam_common.h>
+
+#ifdef CFG_NXP_CAAM_BLOB_DRV
+/*
+ * Initialize the MKVB module
+ *
+ * @ctrl_addr   Controller base address
+ */
+enum caam_status caam_blob_mkvb_init(vaddr_t baseaddr);
+#else
+static inline enum caam_status caam_blob_mkvb_init(vaddr_t baseaddr)
```

Comment:
add `__unused` after `baseaddr`.

You should try a build with `CFG_NXP_CAAM_BLOB_DRV=n` to test.